GUIDELINES.md

```diff
@@ -0,0 +1,70 @@
+Spider writing guidelines
+=========================
+
+Before start working on a spider, read carefully the following guidelines that will make the project more consistent and your PR more probable to be accepted.
+
+Reading code of existing spiders will also help you to understand how to develop.
+
+Spider class
+------------
+
+- All spiders must inherit from `BaseGazetteSpider`
+- Class name must follow the pattern `<State><CityName>Spider` (for example `SpSaoPauloSpider` or `MtFelizNatalSpider`)
+- Spider `name` must follow the pattern `<state>_<city_name>` (for example `sp_sao_paulo` or `mt_feliz_natal`)
+- Add `TERRITORY_ID` attribute with the code found in `territories.csv` file
+- Add `start_date` attribute containing the date of the first gazette available
+
+Example initial spider:
+```
+from datetime import date
+from gazette.spiders.base import BaseGazetteSpider
+
+class SpJundiaiSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3525904"
+    name = "sp_jundiai"
+    allowed_domains = ["jundiai.sp.gov.br"]
+    start_urls = ["https://imprensaoficial.jundiai.sp.gov.br/"]
+    start_date = date(2000, 1, 1)
+
+    def parse(self, response):
+        # Here we have all parsing logic
+```
+
+Field definitions
+-----------------
+
+These are the information that we need to extract for each gazette:
+
+- **date (datetime.date)**: publication date
+- **edition_number (string)**: edition number of gazette
+- **is_extra_edition (boolean)**: some gazettes are extra editions. You can identify them when they have words like *Extra*, *Extraordinário*, *Suplemento*, etc. If unable to identify, return as `False`
+- **territory_id (string)**: value of `TERRITORY_ID` spider attribute
```

Comment:
Would be nice to mention that this field is mandatory. Also, what do you think of mentioning that if no start date is informed is understood that all gazettes will be collected?

Reply:
All fields are mandatory (except edition_number). I will add a note about that.
About start_date, you are right! It will be good to mention that.